src/main/cli/cli.c

```diff
@@ -1326,6 +1359,12 @@ static void cliSerialPassthrough(char *cmdline)
     if (ports[0].baud == 0 && ports[1].id == SERIAL_PORT_USB_VCP) {
         enableBaudCb = true;
     }
+    
+    if (esc_sensor_passthrough) {
+        serialPortUsage_t *passThroughPortUsage = findSerialPortUsageByIdentifier(ports[0].id);
+        passThroughPortUsage->function = FUNCTION_NONE;
```

Comment:
Not sure here, but poor usage was probably intended to be read-only. Modifying it may be ok if passthrough is exited with reset, otherwise it may cause very nasty problems ...

Reply:
as far as i undesrtood the serialpassthrough function it is meant to work untill reset. no matter if the reset comes because of a power cycle or DTR... if i dont free the esc_sensor port like that it sometimes dont open it...

Reply:
OK, then add comment that state is broken and reset must be used to exit. The code may change in future, allowing exist from passthrough back to CLI ...

Reply:
ok added a comment

Reply:
Why do we need to modify `passThroughPortUsage` in the first place?

Reply:
it was because if the ESC sensor function is enbaled, the port is already in use and there is for now no way to change the options. i tried today very hard  ;) but in the end it could only work if the changes get used down to GPIO init because of pullup/down and PP/OD things.
and i dont feel well with changing all of that ..